asf_core_data/pipeline/preprocessing/data_cleaning.py

```diff
@@ -507,9 +511,32 @@ def clean_epc_data(df):
     """
 
     df = remove_empty_features(df)
+
     df = standardise_unknowns(df)
     df = standardise_features(df)
     df = standardise_dates(df)
     df = custom_clean_features(df)
 
     return df
+
+
+def reformat_postcode(df, postcode_var_name="POSTCODE", white_space="remove"):
```

Comment:
this would probably fit a more general utils file, but it can also stay here for now.

Reply:
Agreed, we have move that eventually.